src/workflows/airqo_etl_utils/airqo_gx_utils.py

```
@@ -1,18 +1,21 @@
+from typing import Dict, Any, List
+import os
+from pathlib import Path
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)